### ISSUE TITLE
Source Airtable - Error after Upgraded to 1.0.1

### ISSUE THREAD

**Issue by ans-4175** [airbytehq/airbyte#21281]
## Environment
- **Airbyte version**:  0.40.16
- **OS Version / Instance**: AWS EC2
- **Deployment**: Docker
- **Source Connector and version**: Airtable 1.0.1
- **Destination Connector and version**: Postgres 1.0.36
- **Step where error happened**: Refresh Schema

## Current Behavior
After upgraded Airtable Connector to 1.0.1, I got error of when trying to refresh source schema
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/3961872/211950587-73077455-c762-4c4e-beed-07cd3a6b1d0f.png">
Then I checked for connection test, it was returning
<img width="302" alt="image" src="https://user-images.githubusercontent.com/3961872/211950432-5d3b66a3-9d0b-494b-a470-23476d3cb860.png">

## Expected Behavior
Should have no error refresh schema and connector test

## Logs
<img width="936" alt="image" src="https://user-images.githubusercontent.com/3961872/211950795-5ad347cf-30af-453d-82ac-90493783644e.png">
<img width="928" alt="image" src="https://user-images.githubusercontent.com/3961872/211950852-a81a5839-53ba-43e6-a5e4-f0b5ed51f681.png">

## Steps to Reproduce
1. Upgrade Airtable Source to 1.0.1
2. Refresh Source Schema
3.

## Are you willing to submit a PR?
N/A

**Comment by NumberPiOso** [comment]
I am having API KEY authentication problems with Airtable connector as well when setting up the source

Environment
Airbyte version: 0.40.7
OS Version / Instance: AWS EC2
Deployment: Docker
Source Connector and version: Airtable >=1.0.0
Step where error happened: Set up source


In version 1.0.0 and 1.0.1 I get the following error.
`'404 Client Error: Not Found for url: https://api.airtable.com/v0/meta/bases'`

In version 2.0.0 I get the following one
`ValueError("This authenticator expects a value under the ('credentials', 'access_token') field path. Please check your configuration structure or change the access_token_config_path value at initialization of this authenticator.")`

**Comment by markns** [comment]
Hey @NumberPiOso I'm having the same issue. Did you manage to solve this?

**Comment by NumberPiOso** [comment]
@markns  They fixed it in version **2.0.1** and it is now working for me!

**Comment by sajarin** [comment]
Closing this, feel free to make a new issue if there's an issue :)

**Comment by lemaadi** [comment]
Hi there,

After upgrading to version `2.0.3` we are unable to sync and facing this issue : 
```
2023-03-12 12:23:02 INFO i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/source-airtable:2.0.3 exists...
2023-03-12 12:23:02 INFO i.a.c.i.LineGobbler(voidCall):149 - 
2023-03-12 12:23:02 INFO i.a.c.i.LineGobbler(voidCall):149 - airbyte/source-airtable:2.0.3 was found locally.
2023-03-12 12:23:02 INFO i.a.w.p.DockerProcessFactory(create):130 - Creating docker container = source-airtable-check-783-0-peioh with resources io.airbyte.config.ResourceRequirements@87011e4[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=] and allowedHosts io.airbyte.config.AllowedHosts@2150174d[hosts=[api.airtable.com, airtable.com, *.datadoghq.com, *.datadoghq.eu, *.sentry.io],additionalProperties={}]
2023-03-12 12:23:02 INFO i.a.w.p.DockerProcessFactory(create):175 - Preparing command: docker run --rm --init -i -w /data/783/0 --log-driver none --name source-airtable-check-783-0-peioh --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e STRICT_COMPARISON_NORMALIZATION_WORKSPACES= -e WORKER_CONNECTOR_IMAGE=airbyte/source-airtable:2.0.3 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e STRICT_COMPARISON_NORMALIZATION_TAG=strict_comparison2 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=0 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.42.0 -e WORKER_JOB_ID=783 airbyte/source-airtable:2.0.3 check --config source_config.json
2023-03-12 12:23:02 INFO i.a.w.i.VersionedAirbyteStreamFactory(create):107 - Reading messages from protocol version 0.2.0
2023-03-12 12:23:04 ERROR i.a.w.i.DefaultAirbyteStreamFactory(internalLog):164 - {"error":{"type":"NOT_FOUND"}}
2023-03-12 12:23:04 ERROR i.a.w.i.DefaultAirbyteStreamFactory(internalLog):164 - Check failed
2023-03-12 12:23:05 INFO i.a.w.g.DefaultCheckConnectionWorker(run):120 - Check connection job received output: io.airbyte.config.StandardCheckConnectionOutput@55a95ac[status=failed,message='404 Client Error: Not Found for url: https://api.airtable.com/v0/meta/bases']
2023-03-12 12:23:05 INFO i.a.w.t.TemporalAttemptExecution(get):169 - Stopping cancellation check scheduling...
2023-03-12 12:23:05 INFO i.a.c.i.LineGobbler(voidCall):149 - 
2023-03-12 12:23:05 INFO i.a.c.i.LineGobbler(voidCall):149 - ----- END CHECK -----
2023-03-12 12:23:05 INFO i.a.c.i.LineGobbler(voidCall):149 - 
2023-03-12 12:23:06 INFO i.a.v.j.JsonSchemaValidator(test):129 - JSON schema validation failed. 
errors: $.access_token: is missing but it is required, $.refresh_token: is missing but it is required
2023-03-12 12:23:06 INFO i.a.v.j.JsonSchemaValidator(test):129 - JSON schema validation failed. 
errors: $.private_key: is missing but it is required
2023-03-12 12:23:06 INFO i.a.v.j.JsonSchemaValidator(test):129 - JSON schema validation failed. 
errors: $.method: does not have a value in the enumeration [Standard]
2023-03-12 12:23:06 INFO i.a.c.f.EnvVariableFeatureFlags(autoDisablesFailingConnections):33 - Auto Disable Failing Connections: false
```

How can we fix it ? or how can downgrade to version `2.0.1` as suggested by @NumberPiOso 

Thank you
